README.md

```diff
@@ -191,6 +191,28 @@ print(sa_result.get_table())
 +----------------+-----------+--------------+----------------+------------+-------+------------+---------------------+-----------------+-------+------+
 ```
 
+#### To run with hades2
+Download [Hades2](https://rte-france.github.io/hades2/index.html)
+
+Create a `config.yml` under `$HOME/.itools/`
+```yaml
+load-flow:
+    default: Hades2
```

Comment:
I thinks this is not necessary as on Java side provider will be found using find(...)